packages/electrode-archetype-react-app-dev/lib/webpack-middleware.js

```diff
@@ -255,16 +240,7 @@ class Middleware {
           if (userReporter) userReporter(middlewareOptions, reporterOptions);
           this.webpackDev.lastReporterOptions = reporterOptions;
         };
-
-        transferIsomorphicAssets(this.devMiddleware.fileSystem, err => {
-          // reload assets to activate
-          if (err) return update();
-
-          return isomorphicExtendRequire.loadAssets(err2 => {
-            if (err) console.error("reload isomorphic assets failed", err2);
-            update();
-          });
-        });
+        isomorphicExtendRequire.loadAssets(() => update());
```

Comment:
while no need to transfer assets file, still need to verify that `.isomorphic-loader-config.json` is updated.  see master branch.